### PR TITLE
[QoF] Adding feature that automatically rotate player view when find a item/tile entity successfully

### PR DESCRIPTION
**Pull request by gitboy14**
fintit

### CODE REVIEW

**Comment by Caedis** [comment]
Please have this be under a config option, default false

File: src/main/java/com/gtnh/findit/service/blockfinder/ClientBlockFindService.java [hunk 1]
```diff
@@ -25,6 +29,17 @@ public ClientBlockFindService() {
     public void handleResponse(EntityClientPlayerMP player, BlockFoundResponse response) {
         player.closeScreen();
         ParticlePosition.highlightBlocks(player.worldObj, response.getPositions());
+        lookAtTarget(player, response);
+    }
+
+    public void lookAtTarget(EntityClientPlayerMP player, BlockFoundResponse response){
```
Comment (Caedis) [inline src/main/java/com/gtnh/findit/service/blockfinder/ClientBlockFindService.java]: Consider moving this to a util class so its not duplicated between these this class and `ClientItemFindService`

**Review by Caedis — CHANGES_REQUESTED** [review]

**Comment by gitboy14** [comment]
> Please have this be under a config option, default false

Anything wrong?

**Comment by Caedis** [comment]
The code itself looks fine to me, I just want some other devs to take a look

**Comment by slprime** [comment]
Is it possible to make it always show in the center of the block, not around?

Minecraft1 7 102024-02-1619-11-51

**Review by slprime — APPROVED** [review]

**Review by Caedis — APPROVED** [review]

**Comment by Dream-Master** [comment]
I will change the config to true as default in gtnh otherwise no one will find this nice feature.

**Comment by Caedis** [comment]
Was actually thinking about that too

**Comment by gitboy14** [comment]
> Is it possible to make it always show in the center of the block, not around?
> 
> Minecraft1 7 102024-02-1619-11-51 [ Minecraft1 7 102024-02-1619-11-51 ](https://private-user-images.githubusercontent.com/31038811/305491515-120325c2-cd0d-4c90-a615-cff835f84fc9.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxNDg2ODEsIm5iZiI6MTcwODE0ODM4MSwicGF0aCI6Ii8zMTAzODgxMS8zMDU0OTE1MTUtMTIwMzI1YzItY2QwZC00YzkwLWE2MTUtY2ZmODM1Zjg0ZmM5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE3VDA1Mzk0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0MTNiNTg2Zjc2NTEwNzg2MzlkNmFhNTlmYmU0NjQwOTgwYmI5YmIyZDM1ZmZiMzYwYjk0ZjA5ZDM1YWQ3ZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hCie_7s8Y1kUD4OiG6FaOYep1StIMn9Y3SwRDEwXEPY) [ ](https://private-user-images.githubusercontent.com/31038811/305491515-120325c2-cd0d-4c90-a615-cff835f84fc9.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxNDg2ODEsIm5iZiI6MTcwODE0ODM4MSwicGF0aCI6Ii8zMTAzODgxMS8zMDU0OTE1MTUtMTIwMzI1YzItY2QwZC00YzkwLWE2MTUtY2ZmODM1Zjg0ZmM5LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE3VDA1Mzk0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0MTNiNTg2Zjc2NTEwNzg2MzlkNmFhNTlmYmU0NjQwOTgwYmI5YmIyZDM1ZmZiMzYwYjk0ZjA5ZDM1YWQ3ZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hCie_7s8Y1kUD4OiG6FaOYep1StIMn9Y3SwRDEwXEPY)

Sure, now it works fine.

**Comment by YannickMG** [comment]
A bit late but as a suggestion, how about making it so subsequent presses of the keybind have you focus on the next find in range, letting you cycle through them?

**Comment by AnrDaemon** [comment]
Also, if a rotation could not be instant, that'd also help.

**Pull request merged**